unitTests/utils/ResultListUtilsTest.ts

```diff
@@ -10,9 +10,10 @@ export const ResultListUtilsTest = () => {
     const utils = ResultListUtils;
     let root: SearchInterface;
     let cmp: Component;
+    let resultList: Component;
```

Comment:
I find limiting the number of variables holding the state of the test suite makes it easier to reason about the tests. and avoid mistakes.

Would it be possible to access the result list via the `root` component instead?

Reply:
It was already created as a mock line 15. I just reuse it down the line.